crates/holochain/src/core/workflow/publish_dht_ops_workflow.rs

```diff
@@ -316,59 +323,17 @@ mod tests {
         });
     }
 
-    /// There is a test that shows that if the receipt is set to complete
```

Comment:
The other existing tests are ignored by @neonphog with a comment about re-writing them using the mock network. Is that the approach I've used for the new tests for something else?

I think there might still be some value in those tests because they have a wider scope than the new tests but if we are going to stub the network completely then I've pretty much covered what the workflow does with the new tests and I think the ignored tests can go

Reply:
@ThetaSinner - Mmm, yeah, I think there was some flakiness that had me write that message. I feel like we should just delete them, they weren't written for a particular regression or anything.